catalog/app/containers/Bucket/File.js

```diff
@@ -521,7 +498,7 @@ export default function File({
                 {
                   Ok: ({ ui: { blocks } }) => (
                     <>
-                      {blocks.code && <Code>{code}</Code>}
+                      {blocks.code && <DirCodeSamples {...{ bucket, path }} />}
```

Comment:
it's quite confusing tbh. why call the component `DirCodeSamples` and then use it in both file and dir contexts with an additional `isDirectory` flag?